src/backend/looped/self/super_hero_fly.cpp

```diff
@@ -249,7 +257,7 @@ namespace big
 			PAD::DISABLE_CONTROL_ACTION(0, (int)ControllerInputs::INPUT_COVER, false);
 
 			if (m_timer)
-				m_duration = std::chrono::duration_cast<std::chrono::milliseconds> (std::chrono::system_clock::now() - m_time_stamp);
+				m_duration = std::chrono::system_clock::now() - m_time_stamp;
```

Comment:
Why? If you're worried about accuracy being lost in calculations between calls, you can keep a std::chrono::high_resolution_clock type, and then set the m_duration to a std::chrono::duration_cast at the last moment.